src/app/campaign/availability/goal-form.component.ts

```diff
@@ -31,20 +31,10 @@ export class GoalFormComponent implements OnInit, OnDestroy {
   set flight(flight: Flight) {
     if (flight) {
       this._flight = flight;
-      this.updateGoalForm({ totalGoal: this._flight.totalGoal });
+      this.updateGoalForm({ totalGoal: this._flight.totalGoal, dailyMinimum: this._flight.dailyMinimum });
     }
   }
-  // tslint:disable-next-line
-  private _dailyMinimum: number;
-  get dailyMinimum(): number {
-    return this._dailyMinimum;
-  }
-  @Input()
-  set dailyMinimum(dailyMinimum: number) {
-    this._dailyMinimum = dailyMinimum;
-    this.updateGoalForm({ dailyMinimum });
-  }
-  @Output() goalChange = new EventEmitter<{ flight: Flight; dailyMinimum: number }>();
+  @Output() goalChange = new EventEmitter<Flight>();
```

Comment:
Succinct - i like it!  Will also scale if we add other "goal-ish" fields over there in the future.